zipline/algorithm.py

```diff
@@ -383,6 +383,8 @@ def __init__(self, *args, **kwargs):
 
         self.benchmark_sid = kwargs.pop('benchmark_sid', None)
 
+        self.capital_changes = kwargs.pop('capital_changes', {})
```

Comment:
Could we put a comment either here or in the class docstring of the expected types of the key/value pairs? (I think it was mentioned in PR, but would be beneficial for quick reference here.)